microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -118,41 +134,42 @@ By default, the **Page** tab groups resources by directory.  To display the reso
 
```

Comment:
Change from:
```
By default, the **Page** tab groups resources by directory. To display the resources for each domain as a flat list, instead of grouping them by directory:
```
to:
```
By default, the **Page** tab in the **Sources** tool groups resource files by folder.  To instead display all the resource files for each domain grouped together in a single alphabetized list:

1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.

1. In DevTools, on the main toolbar, select the **Sources** tab.  If that tab isn't visible, click the **More tabs** (![The More tabs icon](../media/more-tabs-icon-light-theme.png)) button, or else the **More Tools** (![More Tools icon.](../media/more-tools-icon-light-theme.png)) button.
```

Reply:
verified